packages/contracts/src/dollar/core/CreditRedemptionCalculator.sol

```diff
@@ -18,7 +18,7 @@ contract CreditRedemptionCalculator is ICreditRedemptionCalculator {
     modifier onlyAdmin() {
         require(
             manager.hasRole(manager.INCENTIVE_MANAGER_ROLE(), msg.sender),
-            "CreditCalc: not admin"
+            "UARCalc: not admin"
```

Comment:
```suggestion
            "CreditCalculator: not admin"
```

Looks like the old naming convention was correct.

Reply:
done